themes/morphosource/css/morphosource.css

```diff
@@ -704,14 +800,195 @@ styles for original hp layout
-	line-height:1.2em;
+	line-height:1.4em;
+	position:relative;
+}
+
+#dashboardMedia .projectMedia {
+	height: 190px;
+}
+
+#dashboardMedia .projectMedia .entityBatchCheckContainer{
+	position:absolute;
+	top:5px;
+	right:5px;
+}
+
+#dashboardMediaHeader {
+	background-color: #003880;
+	padding-left: 10px;
+	padding-right: 10px;
+	height: 50px;
+	border-radius: 5px;
+	-moz-border-radius: 5px; 
+	-webkit-border-radius: 5px;
+}
+
+#dashboardMediaHeader:before {
+  content: "";
+  display: inline-block;
+  vertical-align: middle;
+  height: 100%;
+}
+
+.dashboardMediaHeaderItem {
+	display: inline-block;
+	vertical-align: middle;
+}
+
+.dashboardMediaHeaderSelect {
+	margin-right: 10px;
+}
+
+.batchTopButton {
+	margin-left: 5px;
+	margin-right: 5px;
+	margin-top: 2px;
 }
+
+.treeSortButtons {
+	margin-top: 15px;
+}
+
+#entityBatchPaneContainer{
+	border: 1px solid #003880;
+	box-shadow: 0 0px 3px #828282;
+	padding: 10px;
+	/*height: 460px;*/
+}
+
+#entityBatchPaneContainer .formLabel {
+	font-weight: normal;
+	font-family: 'neuzeitlgt', sans-serif;
+	font-size: 13px;
+	margin-left: 5px;
+}
+
+#entityMediaBatchFormContainer {
+	height: 400px;
+}
+
+#batchColLeft {
+	float: left;
+	width: 415px;
+}
+
+#batchColRight {
+	float: right;
+	width: 415px;
+}
+
+#batchButtonContainer{
+	height: 60px;
+}
+
+.batchButton {
+	width: 70px;
+}
+
+#entityMediaBatchEditContainer {
+	margin-top: 15px;
+	margin-bottom: 15px;
+}
+
+#entityView {
+	margin-top: 15px;
+	margin-bottom: 15px;
+}
+
+.marginTop10Px {
+	margin-top: 10px;
+}
+
+.entityListItem {
+	min-height: 40px;
+}
+
+.entityListItemFooter {
+	text-align: center;
+}
+
+.entityTileContainer {
+	height: 320px !important;
+}
+
+.entityListContainer{
+	margin-left: 5px;
+	margin-right: 5px;
+	margin-bottom: 10px;
+}
+
+.entityListCell {
+	padding-left: 5px;
+	padding-right: 5px;
+}
+
+.entityGroup {
+	overflow: auto;
+}
+
+.entityGroup .projectMediaContainer {
+	padding-left: 10px !important;
+	padding-right: 10px !important;
+}
+
+.entityNestGroupContainer {
+	overflow: auto; 
+	border-radius: 5px; 
+	-moz-border-radius: 5px; 
+	-webkit-border-radius: 5px; 
+	margin-bottom: 10px
+}
+
+.entityNestSpeciesGroup {
+	border-top: 1px solid #689899;
+	border-left: 1px solid #689899;
+	border-right: 1px solid #689899;
+	border-radius: 5px;
+    -moz-border-radius: 5px;
+    -webkit-border-radius: 5px;
+}
+
+.entityGroupLabel {
+	padding-left: 10px;
+	padding-right: 10px;
+	height: 25px;
+	border-radius: 5px;
+	-moz-border-radius: 5px; 
+	-webkit-border-radius: 5px;
+}
+
+.entityGroupLabel:before {
+    content: "";
+    display: inline-block;
+    vertical-align: middle;
+    height: 100%;
+}
+
+.entityGroupLabelB{
+	display: inline-block;
+	width: 200px;
+	height: 200px;
+	background-color: #C7D3E3;
+	border: 1px solid #689899;
+	border-radius: 5px;
+	-moz-border-radius: 5px; 
+	-webkit-border-radius: 5px;
+}
+
+.entityGroupLabelItem {
+	display: inline-block;
+	vertical-align: middle;
+	/*color: #fff;*/
+}
+
```

Comment:
Check all above are used, maybe sort into batch and basic entity viewer sections